app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -318,23 +385,30 @@ class BrowserTabViewModel(
 
     override fun loadingStarted(url: String?) {
         Timber.v("Loading started")
+        if (!currentBrowserViewState().browserShowing) return
         val progress = currentLoadingViewState()
         loadingViewState.value = progress.copy(isLoading = true)
         pendingUrl = url
         site = null
         onSiteChanged()
     }
 
-    override fun navigationOptionsChanged(navigationOptions: BrowserNavigationOptions) {
+    override fun navigationOptionsChanged(navigation: BrowserNavigationOptions) {
```

Comment:
if changing the param name, should probably change it in the super class too (lint warning)